tests/test_compendia.py

```diff
@@ -25,8 +25,8 @@
         "sha1": "3c5de2dba42c2768e7542a0262d2b9a1e9fc9f45",
         "s3_bucket": "test-bucket",
         "s3_key": "test-compendia.zip",
-        "created_at": "2020-10-02:00:00.0Z",
-        "last_modified": "2020-10-02:00:00.0Z"
+        "created_at": "2020-10-02T00:00:00Z",
+        "last_modified": "2020-10-02T00:00:00Z"
```

Comment:
What happened here? Did these get mangled at some point? What format does the API return?

Reply:
yeah, I think it edited them to be the day that I originally wrote the tests for some reason.
The api returns:
```json
"created_at":"2019-11-18T23:27:16.710087Z",
"last_modified":"2020-01-23T20:22:33.465647Z"
```
so I must've messed up